spec/acceptance/linux_spec.rb

```diff
@@ -4,68 +4,53 @@
 # Red-Hat 6 is the only platform we cannot reliably perform package actions on
 redhat_six = os[:family] == 'redhat' && os[:release].to_i == 6
 windows = os[:family] == 'windows'
+inventory_hash = inventory_hash_from_inventory_file
+inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'ssh_nodes')
+inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'winrm_nodes')
+inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'docker_nodes')
+write_to_inventory_file(inventory_hash, 'inventory.yaml')
 
 describe 'linux package task', unless: redhat_six || windows do
-  include Beaker::TaskHelper::Inventory
-  include BoltSpec::Run
-
-  def bolt_config
-    { 'modulepath' => RSpec.configuration.module_path }
-  end
-
-  let(:bolt_inventory) { hosts_to_inventory.merge('features' => ['puppet-agent']) }
-
   describe 'install action' do
     it 'installs rsyslog' do
-      apply_manifest_on(default, "package { 'rsyslog': ensure => absent, }")
-      result = run_task('package::linux', 'default', 'action' => 'install', 'name' => 'rsyslog')
-      expect(result[0]).to include('status' => 'success')
-      expect(result[0]['result']).to include('status' => %r{install})
-      expect(result[0]['result']).to include('version')
+      apply_manifest("package { 'rsyslog': ensure => absent, }")
+      result = run_bolt_task('package::linux', 'action' => 'install', 'name' => 'rsyslog')
+      expect(result.exit_code).to eq(0)
+      expect(result.stdout).to match(%r{install})
     end
 
     it 'errors gracefully when bogus package requested' do
-      result = run_task('package::linux', 'default', 'action' => 'install', 'name' => 'foo')
-      # older EL platforms may report that the bogus package is uninstalled,
-      if result[0]['status'] == 'failure'
-        expect(result[0]['result']).to include('status' => 'failure')
-        expect(result[0]['result']['_error']).to include('msg')
-        expect(result[0]['result']['_error']).to include('kind' => 'bash-error')
-        expect(result[0]['result']['_error']).to include('details')
-      elsif result[0]['status'] == 'success'
-        expect(result[0]['result']).to include('status' => 'uninstalled')
-      else
-        raise "Unexpected result: #{result}"
-      end
+      expect { run_bolt_task('package::linux', 'action' => 'install', 'name' => 'foo') }.to raise_error(RuntimeError)
     end
   end
 
   describe 'status action' do
     it 'status rsyslog' do
-      apply_manifest_on(default, "package { 'rsyslog': ensure => present, }")
-      result = run_task('package::linux', 'default', 'action' => 'status', 'name' => 'rsyslog')
-      expect(result[0]).to include('status' => 'success')
-      expect(result[0]['result']).to include('status' => %r{install})
-      expect(result[0]['result']).to include('version')
```

Comment:
should we still check for version ?

Reply:
I think so, If this check was removed due to a different upgrade target it could be the case that a newer version is not available, in which case we may need to find a different package rather than `rsyslog`. I have also thought about trying to intentionally downgrade a package and then upgrade, but that got messy...

Reply:
@tphoney @donoghuc Thanks for the valuable comments. Have made changes to litmus function to handle the version too. Will get the PR up for review soon. Tested locally and all looks good.